ansible_collections/arista/avd/roles/eos_designs/doc/fabric-variables.md

```diff
@@ -14,13 +14,24 @@
   - IBGP (only with OSPF or ISIS variants in underlay)
 - Only summary network addresses need to be defined. IP addresses are then assigned to each node, based on its unique device id.
   - To view IP address allocation and consumption, a summary is provided in the auto-generated fabric documentation in Markdown and CSV format.
-- The variables should be applied to all devices in the fabric.
 
 *Only supported with core_interfaces data model.
 
+## Flagging a Device as Not Deployed
+- It is possible to provision configurations for a complete topology but flag devices as undeployed using the host level variable `is_deployed: false`. I.e. via the Ansible inventory file.
+- By default, this will have no impact, except cause the device to be ignored by CloudVision when used in conjunction with the `eos_config_deploy_cvp` role. If the device is not present in the network, `eos_validate_state` role will fail tests on peers of the undeployed device trying to verify that interfaces are up.
+- To overcome this and shutdown interfaces towards undeployed peers, the variable `shutdown_interfaces_towards_undeployed_peers` can be used. This will override any setting that enables a port across the fabric and will satisfy the `eos_validate_state` role interface tests.
+
+Note: The variables should be applied to all devices in the fabric.
+
+
+
 **Variables and Options:**
 
 ```yaml
+# Shutdown interfaces on devices that face undeployed peers.
```

Comment:
please add the `is_deployed: < true | false | default -> true >` as well. It is only documented inside another role.

Reply:
```is_deployed``` is a host level variable and is not a fabric variable so I don't think it should be explained here. 

Although I have added an explanation in it's own section above.

Reply:
@perimore - let's still add the variable somewhere in `eos_designs` role (and indicate that it is used as well in the `eos_config_deploy_cvp` to make it clear that it can be used by `eos_designs` in conjunction with the new knob you are adding otherwise it can be a bit confusing. I think it can be enough to just add the variable clearly to the paragraph you wrote above:

```
# Use at the host level
is_deployed: < true | false | default -> true >
```

Reply:
I've added the variable as below to "Variables and Options:". Is this enough? `is_deployed` is explained in the paragraph.

# Use at the host level to flag a device as undeployed. See the explanation above as to how this is used in combination with `shutdown_interfaces_towards_undeployed_peers` below.
is_deployed: < true | false | default -> true >

Reply:
I think for clarity's sake I would leave this "special" variable definition in the section where you explain it - to be clear that this is not one that should be applied to the whole Fabric